com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/context.py

```diff
@@ -450,3 +453,4 @@ class ConfigParams:
     """
     VCAP_INFO = 'topology.service.vcap'
     VCAP_NAME = 'topology.service.name'
+    REMOTE_SUBMIT = 'topology.remote_submit'
```

Comment:
Maybe `topology.forceRemoteBuild` since that is the portion that is being forced, the submit is always to a remote system.